src/movie_lib.cc

```diff
@@ -99,7 +99,7 @@ static void _nfPkDecomp(unsigned char* buf, unsigned char* a2, int a3, int a4, i
 
 static constexpr uint16_t loadUInt16LE(const uint8_t* b);
 static constexpr uint32_t loadUInt32LE(const uint8_t* b);
-static uint8_t getOffset(uint16_t v);
+static int getOffset(uint16_t v);
```

Comment:
The change in this file seems to be related to the #188.

Reply:
Yes, I didn't realise that pushing an unrelated commit (for fixing the movies) on my fork's master branch would add it to this pull request. I should undo that.

Reply:
Ok I'm quite new to this tbh... should I "revert" (which will add another commit undoing the unrelated changes) or should I force push to remove the unwanted commit from my repository? I might have done the latter if nobody else was looking at this yet but I don't want to cause problems by messing with history for other people. Or have I made a mess of this pull request and should trash it and start over?